main.go

```diff
@@ -167,21 +178,29 @@ func main() {
 //   3. Run data conversion
 //   4. Generate report
 func toSpanner(driver, projectID, instanceID, dbName string, ioHelper *ioStreams, outputFilePrefix string, now time.Time) error {
```

Comment:
We're developing an inconsistency in the way that we use vars like driverName, schemaOnly, dataOnly and filePrefix.
Some we only access in the main function, and pass as function args e.g. driverName.
But for others, we access directly in code outside of main.

We already had this inconsistency before this PR, but adding dataOnly and schemaOnly expands it a bit.

What do you think about the following convention:
- any access of vars in main.go are direct accesses
- we no longer pass vars as args to functions

That would mean dropping the driver and outputFilePrefix args for toSpanner.

Reply:
Sounds good to me. The integration tests may need to be refactored accordingly.

Reply:
@nevinheintze @hengfengli Agreed. For consistency, I will make changes as suggested and also update the integration tests. I will raise a separate PR for that very soon. Let's go ahead and merge the current one, if that's fine.

Reply:
Yes, separate PR is good!